components/button-cards/solution-card/solution-card.js

```diff
@@ -0,0 +1,48 @@
+const cardRows = document.querySelectorAll('.card-row')
```

Comment:
We are reusing this code a lot, maybe we should move the code into card.js to have a more DRY approach

Reply:
An alternative would be to use what I have used in the category cards and contact cards. That code also gives the overall card component matching heights to their siblings in the row, based on the inner content